modules/@apostrophecms/ui/index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   options: {
-    alias: 'ui'
+    alias: 'ui',
+    widgetMargin: '0'
```

Comment:
this will is a CSS value, so string is fine, allows for a CSS'y format like `20px 0`